### DISCUSSION TITLE
Is curseforge really a massive pile of sh or am I just doing something wrong?

### DISCUSSION THREAD

**Discussion by ThatCoffeeGuy** [itzg/docker-minecraft-server#2500] [Q&A] [answered]
So I've been trying to make https://www.curseforge.com/minecraft/modpacks/roguelike-adventures-and-dungeons-2 work for a few hours now.

I've tried the old method of installing it, didn't work.

I am trying to new method new, with API keys. Sometimes my IP gets blocked to the API server, sometimes it works. Now I learn that some modmakers love to disable auto-download (so out of 300 modmakers, there's 3 who want to make it harder to install their mods)

I collect them, put them into the mods folder, 2 of 3 works, one remains missing, It's in the mods folder, proper version and everything, yet it's not being recognized. Restarting the server I get another API ban for 10 minutes.

I tried like 4 different versions of java as well, including the one it supposedly supports (1.17> java8-multiarch)

Is this the regular experience or am I just being unlucky?

**Comment by itzg** [comment]
At this point the manually downloaded mods need to placed into a volume mounted at `/downloads` as described here

https://docker-minecraft-server.readthedocs.io/en/latest/types-and-platforms/mod-platforms/auto-curseforge/#manual-downloads

**Reply by ThatCoffeeGuy** [reply, marked as answer]
Thank you for the answer!

But it also states that /mods is checked, so I'd assume I did everything right, since 2 of the 3 manually downloaded mods picked up properly.  

I think the mod in question was BetterDefaultBiomes or something like this  - I downloaded it the same way I did with the other 2, but this one never got picked up by the server.

edit: I ended up deleting this awful mess so I can't provide a precise modname or logs. The ratelimited API that bans with normal usage, the authors who decide their mods can't be autodownloaded, the tons of errors and random issues...  I'll stick to FTB, this is a bad joke. 

As always, I appreciate your answer and work on the image.
